src/main/java/de/retest/recheck/MavenConformFileNamerStrategy.java

```diff
@@ -15,13 +15,17 @@ public FileNamer createFileNamer( final String... baseNames ) {
 			@Override
 			public File getFile( final String extension ) {
 				final String baseName = Joiner.on( "/" ).join( baseNames );
-				return new File( "src/test/resources/retest/recheck" + File.separator + baseName + extension );
+				return new File( "src/test/resources/retest/"
+						+ System.getProperty( RECHECK_FOLDER_PROPERTY, RECHECK_FOLDER_DEFAULT ) + File.separator
```

Comment:
The recheck folder is usually a path relative to the project directory, so this wouldn't work?! See recheck-web example config: https://github.com/retest/recheck-web/blob/master/src/test/resources/retest/retest.properties